src/test/java/blitz/gameOfLife2024/GameTest.java

```diff
@@ -0,0 +1,27 @@
+package blitz.gameOfLife2024;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+public class GameTest {
+
+    @Test
+    public void blinker() {
```

Comment:
nextGen()

The name of your test should match the name of the method being tested